west.yml

```diff
@@ -287,7 +287,7 @@ manifest:
       groups:
         - crypto
     - name: mcuboot
-      revision: 898a1ca64a759541d9fcc37fef921db93d99ad70
+      revision: c06f7bb36782e0220b30c3d0640219d1368735d3
```

Comment:
I know I can get to this commit sha but I don't see it when I click on the commits for the repo?

Reply:
It gets pushed to `upstream-sync` for CI to run and approval, once approved and CI has passed, it gets pushed to `main`

Reply:
what is the reason? Is this incase CI fails you don't have main branch with some messed up SHA?